big_tests/tests/muc_SUITE.erl

```diff
@@ -1507,6 +1512,7 @@ admin_member_list_allowed(Config) ->
         %% Kate can't get member list
         check_memberlist(Kate, no, Config),
         %% setup room - allow getmemberlist for moderator
+        escalus:send_and_wait(Alice, stanza_configuration_form_request(?config(room, Config))),
```

Comment:
We should check, that we received what we supposed to receive.
I propose to have a function send_iq_and_wait_for_result/1.
That would:
- check that IQ result id matches IQ request id
- handle error and print both request and error as a readable error.

Motivation: if a service is broken, it would return error service-unavailable and we just continue the test execution.
From time to time there are "unexpected" presences routed in our tests, that also can be consumed by send_and_wait without any assertion.